src/components/homepage/Artists.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import Carousel from "react-elastic-carousel";
+
+import mock from "./../../api/mock.json";
+import { Main, Title, SongTitles, Img, DivImg } from "./Carousel";
+
+const breakPoints = [
+  { width: 1, itemsToShow: 1 },
+  { width: 550, itemsToShow: 2, itemsToScroll: 2 },
+  { width: 768, itemsToShow: 3 },
+  { width: 1200, itemsToShow: 4 },
+];
+
+const TrendArtists = () => {
+  const [data, setData] = useState([]);
+
+  const getMock = () => {
+    setData(mock.items);
+    return mock.items;
```

Comment:
This return does nothing (you don't use the return value in your useEffect code)